schemas/asn1-lib-module-schema.json

```diff
@@ -11,25 +11,28 @@
       "type": "string",
       "pattern": "^[A-Z]([a-zA-Z0-9]|-)*$"
     },
-    "referenceList": {
-      "type": "array",
-      "items": { "$ref": "#/definitions/nonEmptyString" }
-    },
     "fileList": {
       "type": "array",
       "items": { "$ref": "#/definitions/nonEmptyString" }
     },
-    "importedType": {
+    "externalReference": {
       "type": "object",
       "properties": {
-        "from": { "$ref": "#/definitions/asn1TypeName" },
```

Comment:
is asn1TypeName definition still needed?

Reply:
it's not, removed, thanks :)